client/src/connection.ts

```diff
@@ -94,6 +94,16 @@ export function connect() {
     store.dispatch("channels/addMessage", message)
   });
 
+  // eslint-disable-next-line
+  connection.on("message:edit", async (editMessage: ServerEditMessage) => {
```

Comment:
message:update is recommended to follow CRUD conventions

Reply:
done